### PR TITLE
Update the Managing Customer Accounts page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates the Managing Customer Accounts page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/customer-account-manage.html

### CODE REVIEW

File: src/customers/customer-account-manage.md [hunk 1]
```diff
@@ -153,6 +150,9 @@ These columns are available by changing the [column layout]({% link stores/admin
 |Shipping Address|The address where orders are to be shipped.|
 |VAT Number|The value-added tax number that is associated with the customer address. For [digital goods]({% link tax/eu-place-of-supply.md %}) sold in the EU, the VAT is based on the billing address of the customer. <br/><br/>**_Note:_** This field is not the same as the Tax/VAT Number.|
 |Account Lock|Indicates the status of the account. As a security measure, customer accounts can be [locked]({% link customers/password-options.md %}) after too many login attempts. Values: Locked / Unlocked|
+|Status|The current user status. Options: `Active` / `Inactive`|
```
Comment (dobooth) [inline src/customers/customer-account-manage.md:153]:
```suggestion
|Status|The current user status. Options: Active / Inactive|
```
No need to wrap values in backticks. Same below.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/customers/customer-account-manage.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/customers/customer-account-manage.md]: Thanks, done 😊

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**